docs/service-brokers/gcp-broker-service-classes/bigquery/docs/plans-details.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Services and Plans
+type: Details
+---
+
+## Service description
+
+The service provides the following plan names and descriptions:
+
+| Plan Name | Description |
+|-----------|-------------|
+| `Beta Plan` | BigQuery plan for the Beta release of the Google Cloud Platform Service Broker |
+
+## Provisioning parameters
+
+Provisioning an instance creates a new BigQuery dataset. The provisioning parameters are as follows:
+
+| Parameter Name | Type | Description | Required | Default Value |
+|----------------|------|-------------|----------|---------------|
+| **datasetId** | `string` | A user-specified, unique ID for the BigQuery dataset. Must be 1-1024 characters long. Must contain only letters, numbers, or underscores. | YES | - |
+| **defaultTableExpirationMs** | `string` | The default lifetime of all tables in the dataset in milliseconds. The minimum value is `3600000` milliseconds per one hour. Once this property is set, all newly-created tables in the dataset have the **expirationTime** property set to the creation time plus the value in this property. Changing the value only affects new tables, not existing ones. When a given table reaches the **expirationTime**, that table is deleted automatically. If you modify or remove the table's **expirationTime** before the table expires, or if you provide an explicit **expirationTime** when creating a table, that value takes precedence over the default expiration time indicated by this property. | NO |  `3600000` |
```

Comment:
The part: 
`Once this property is set, all newly-created tables in the dataset have the **expirationTime** property set to the creation time plus the value in this property` is a bit unclear - is it the value of this property?

Reply:
Discussed, will explain it a bit.